code/modules/mob/living/carbon/xenomorph/charge_crush.dm

```diff
@@ -207,7 +208,7 @@
 						continue
 					charger.visible_message("<span class='danger'>[charger] runs [victim] over!</span>",
 						"<span class='danger'>We run [victim] over!</span>", null, 5)
-					victim.take_overall_damage(CHARGE_SPEED(src) * 10, blocked = victim.run_armor_check("chest", "melee"))
+					victim.take_overall_damage(CHARGE_SPEED(src) * 10, blocked = victim.run_armor_check("chest", "melee")) //This should not use take_overall_damage, this is a direct damage proc.
```

Comment:
either fix it or remove this comment

Reply:
going to fix in another PR, i'll remove it

Reply:
Still not removed